akka-docs/src/main/paradox/stream/stream-refs.md

```diff
@@ -135,8 +134,8 @@ The process of preparing and running a `SinkRef` powered distributed stream is s
   This is in order to not complicate the mental model what materializing such value would mean.
   
   If you have an use case for building a fan-in operation accepting writes from multiple remote nodes,
-  you can build your Sink and prepend it with a `Merge` stage, each time materializing a new `SinkRef`
-  targeting that Merge. This has the added benefit of giving you full control how to merge these streams 
+  you can build your Sink and prepend it with a `MergeHub` stage, each time materializing a new `SinkRef`
```

Comment:
As this was talking about materializing a new sink ref i assume it meant MergeHub

Reply:
Yes thanks